sdks/java/core/src/test/java/com/google/cloud/dataflow/sdk/transforms/display/DisplayDataMatchers.java

```diff
@@ -16,13 +16,19 @@
 
 package com.google.cloud.dataflow.sdk.transforms.display;
 
+import static org.hamcrest.Matchers.allOf;
```

Comment:
I see a few other matchers classes exposed:

* `DataflowMatchers`
* `TestUtils.KVMatcher`
* `WindowMatchers`

Should these all be moved internal? I can follow-up with another PR.